scripting/get5/teamlogic.sp

```diff
@@ -169,17 +155,17 @@ public void MoveClientToCoach(int client) {
   // coaching command. Otherwise we manually move them to spec
   // and set the coaching target.
   // If in freeze time, we have to manually move as well.
-  if (!InWarmup() && InFreezeTime()) {
+  if (InWarmup()) {
+    LogDebug("Moving %L indirectly to coach slot via coach cmd", client);
```

Comment:
Why is this "indirectly"? Wouldn't it just be "Moved to coach slot with coach command"?

Reply:
I think this is because we are making the client indirectly call `coach`, that's why. We can change it, but this was just what was here before I modified anything.